### PR TITLE
Implement Gravatar support behind config variable.

### PR DESCRIPTION
**Pull request by brenns10**
This branch allows you to set a config variable based on how you would like to use Gravatars. Your options are:

- `always`: Gravatars are used in place of whatever photo is provided in the Employee `photo_url` field.
- `backup`: Gravatars are used whenever an Employee has a blank `photo_url` field.
- anything else: gravatars are disabled (at which point the theme provides a sane default)

### CODE REVIEW

**Comment by brenns10** [comment]
I can add tests for this too. I probably ought to, given that I haven't explored the test code yet :grimacing:

**Comment by sjaensch** [comment]
This is great, thanks @brenns10! A test or two would be nice, then we can merge this in.

**Comment by brenns10** [comment]
Tests have arrived. They focus on the cases for the configuration variable, rather than the correctness of the gravatar URL. The URL is constructed as per the [docs](https://en.gravatar.com/site/implement/images/) and I can anecdotally say that the URLs work great.

File: config-example.py [hunk 1]
```diff
@@ -22,3 +22,9 @@
 # Name of the S3 bucket used to import employee data from a file named employees.json
 # Check out /import/employees.json.example to see how this file should look like.
 S3_BUCKET = 'employees'
+
+# When do we use Gravatar? Options are:
+# * 'always' - prefers Gravatar over the Employee.photo_url
+# * 'backup' - use Gravatar when photo_url is empty
+# * anything else - disabled
+GRAVATAR = 'always'
```
Comment (sjaensch) [inline config-example.py]: I'd vote for 'backup' as the example / default, as I'd expect most people would want to have the company-specific photo override gravatar, if one is available.

File: models/employee.py [hunk 2]
```diff
@@ -78,6 +80,25 @@ def update_from_dict(self, d):
         self.department = d.get('department')
         self.meta_department = get_meta_department(self.department)
 
+    def get_gravatar(self):
+        """Creates gravatar URL from email address."""
+        email = '{user}@{domain}'.format(user=self.username, domain=config.DOMAIN)
```
Comment (sjaensch) [inline models/employee.py]: How about moving this to an email property, similar to how full_name works? Seems pretty useful. Please also add a small test in employee_test.py, which will make sure this line is tested as well.

Comment (brenns10) [inline models/employee.py]: A quick code search shows that the rest of the code seems to use `employee.user.email()` ([eg](https://github.com/Yelp/love/blob/master/logic/love.py#L68)), so I think I'll actually do that instead.

**Comment by brenns10** [comment]
Okay, these last two commits should address your review :)

**Pull request merged**